### ISSUE TITLE
New Explorer; short cut to excute query

### ISSUE THREAD

**Issue by davesmits** [SigNoz/signoz#3269]
## Describe the solution you'd like

In the query builder of the new trace and log explorer there is a great autocomplete to quickly build queries. With enter you accept the suggested value for the key, operator and value of your query. When the query is ready you need to take your hands of the keyboard an click on the 'Run Query' button

It would be nice to have a shortcut, for example ctrl + enter to directly close the auto complete and run the query.

**Comment by pranay01** [comment]
@davesmits Do you mean that after you have selected the filters you should just be able to run the query with keyboard keys, rather than clicking on the `Run Query` button?

**Comment by mustajab-ikram** [comment]
Are we taking this up @pranay01?

**Comment by pranay01** [comment]
@mustajab-ikram You can take this up if interested. Would be good if you can propose an approach here before starting to code.

cc @palashgdev

**Comment by mustajab-ikram** [comment]
@pranay01 @palashgdev  
I've integrated a keyboard event listener for Ctrl + Enter, allowing users to execute queries seamlessly. When triggered, it closes the autocomplete dropdown and runs the query programmatically.
Is this the correct approach?
